### ISSUE TITLE
Multiplexer feature

### ISSUE THREAD

**Issue by IustinMicu** [microsoft/terminal#3155]
# Description of the new feature/enhancement

A **multiplexer** feature would be very useful and cool!
By having multiple panes in a single windows:
 - You can easily copy/paste output from one pane to another.
 - You can work on multiple tasks at once.
 - You can monitor multiple commands at once.
 - You can compose a command in one pane while looking at the 'help' page for that command in another pane.

**Comment by nofunatall** [comment]
What's wrong with tmux or screen?

**Comment by IustinMicu** [comment]
Whats the point of your question @nofunatall ?
I didn't ask for opinions about what application should I use if I need a multiplexer console!
Here we discuss about windows terminal!

**Comment by nofunatall** [comment]
Let me rephrase that.
Are you after just window / tab splitting via hotkeys or are you after all the features of a fully fledged multiplexer?

**Comment by IustinMicu** [comment]
I imagine it's best to start small.
At first it`s enough to have the splitting thing. Then more functionality can be added.

**Comment by DHowett-MSFT** [comment]
This is, broadly, a /dupe of #1000. Thanks.

We actually have split panes today!

![image](https://user-images.githubusercontent.com/14316954/66779632-815d1b80-ee83-11e9-8858-f1467abb3579.png)

**Comment by o-sdn-o** [comment]
Perhaps he had in mind here is a multiplexer

**Comment by o-sdn-o** [comment]
I completely agree that this is not the main feature, since soon it will not need to be implemented inside the Windows Terminal at all, as modern tmuxes will perform this functionality, and this will be cross-platform.

https://youtu.be/5h1fJ-es8kQ
